src/neuroconv/datainterfaces/icephys/baseicephysinterface.py

```diff
@@ -33,6 +26,15 @@ def get_source_schema(cls) -> dict:
         return source_schema
 
     def __init__(self, file_paths: list):
+        # Check if the ndx_dandi_icephys module is available
+        dandi_icephys_spec = importlib.util.find_spec("ndx_dandi_icephys")
+        if dandi_icephys_spec is not None:
+
```

Comment:
```suggestion
        if dandi_icephys_spec is not None:
            from ndx_dandi_icephys import DandiIcephysMetadata
            
            self.DandiIcephysMetadata = DandiIcephysMetadata

```

Reply:
Thanks for fixing this. Went to eat and then got into a meeting.